src/d3d9/d3d9_bridge.h

```diff
@@ -0,0 +1,114 @@
+#pragma once
+
+#include <windows.h>
+#include "../util/config/config.h"
+#include "../util/util_error.h"
+#include "../util/log/log.h"
+
+#include "../vulkan/vulkan_loader.h"
+
+#include <bitset>
+
+// NOTE: You must include "d3d9_include.h" or "d3d8_include.h" before this header.
+
+/**
+ * The D3D9 bridge allows D3D8 to access DXVK internals.

```

Comment:
Second mention has a typo: it says 8 instead of 9

Reply:
There's no typo here. The d3d9 bridge is what d3d8 code in d8vk uses to interact with d3d9 state.